Livecheckables/clojure.rb

```diff
@@ -1,6 +1,6 @@
 class Clojure
   livecheck do
-    url "https://github.com/clojure/brew-install.git"
-    regex(/^v?(\d+(?:\.\d+)+)$/i)
+    url "https://clojure.org/guides/getting_started"
+    regex(/linux-install-v?(\d+(?:\.\d+)+)/i)
```

Comment:
```suggestion
    url "https://raw.githubusercontent.com/clojure/homebrew-tools/master/Formula/clojure.rb"
    regex(/url ".*?clojure-tools-v?(\d+(?:\.\d+)+)\.t/i)
```

If the goal is to keep our formula consistent with the first-party tap formula, then it seems like we can just match the version in their formula, right?

Reply:
yeah, that also works for me.